src/crimson/os/seastore/onode_manager/staged-fltree/node.cc

```diff
@@ -1801,7 +1803,7 @@ LeafNode::erase(context_t c, const search_position_t& pos, bool get_next)
       return eagain_ertr::make_ready_future<Ref<tree_cursor_t>>();
     }
   }).safe_then([c, &pos, this_ref = std::move(this_ref),
-                this, FNAME] (Ref<tree_cursor_t> next_cursor) {
+                this, FNAME] (Ref<tree_cursor_t> next_cursor) mutable {
```

Comment:
ditto.

Reply:
This is needed because `this_ref` will be moved into nested lambda-fn below.